packages/ui5-application-writer/src/index.ts

```diff
@@ -49,22 +55,27 @@ async function generate(basePath: string, ui5AppConfig: Ui5App, fs?: Editor): Pr
 
     // ui5-local.yaml
     const ui5LocalConfigPath = join(basePath, 'ui5-local.yaml');
-    const ui5LocalConfig = await UI5Config.newInstance(fs.read(ui5LocalConfigPath));
-    ui5LocalConfig.addUI5Framework(
-        ui5App.ui5.framework,
-        ui5App.ui5.localVersion,
-        ui5App.ui5.ui5Libs as string[],
-        ui5App.ui5.ui5Theme
-    );
-    ui5LocalConfig.addFioriToolsAppReloadMiddleware();
-
-    // Add optional features
-    await applyOptionalFeatures(ui5App, fs, basePath, tmplPath, [ui5Config, ui5LocalConfig]);
-
-    // write ui5 yamls
+    if (!(ui5AppConfig.app?.projectType === 'CAPJava' || ui5AppConfig.app?.projectType === 'CAPNodejs')) {
```

Comment:
Can we please use the positive case:
```suggestion
    if (projectType ===  'EDMXBackend' ) {
```

Reply:
Made projectType mandatory and included positive case